test/integration/golden_response/get_statvar_group/uk.json

```diff
@@ -1358,211 +1407,6 @@
           "id": "Count_Product_MobileCellularSubscription_AsFractionOf_Count_Person",
           "searchName": "Count Of Product With Product Type Mobile Cellular Subscription As Fraction Of Count Person",
           "displayName": "Count Of MobileCellularSubscription As Fraction Of Count Person"
-        },
```

Comment:
I'm a bit unsure about this...

Reply:
These show up twice in the old cache, and now they are removed in "misc" category. Seems to be a good change.